engine/graphics/src/vulkan/graphics_vulkan_device.cpp

```diff
@@ -1667,6 +1673,16 @@ namespace dmGraphics
             vk_create_pool_info.queueFamilyIndex = (uint32_t) queueFamily.m_GraphicsQueueIx;
             vk_create_pool_info.flags            = VK_COMMAND_POOL_CREATE_RESET_COMMAND_BUFFER_BIT;
             res = vkCreateCommandPool(logicalDeviceOut->m_Device, &vk_create_pool_info, 0, &logicalDeviceOut->m_CommandPool);
+
+            if (res == VK_SUCCESS)
+            {
+                memset(&vk_create_pool_info, 0, sizeof(vk_create_pool_info));
+                vk_create_pool_info.sType            = VK_STRUCTURE_TYPE_COMMAND_POOL_CREATE_INFO;
+                vk_create_pool_info.queueFamilyIndex = (uint32_t) queueFamily.m_GraphicsQueueIx; // Use the same queue for now (use a transfer queue at some point)
```

Comment:
Vulkan supports a dedicated transfer queue just for data uploads, that can run in parallell with the graphics queue. It's a bit more complicated to set that up, but might be worth it at some point.